tests/conftest.py

```diff
@@ -68,7 +68,6 @@ def check_same_image(
             raise OSError(f"No expected output found at {expected}.")
         actual = tmp_path / "actual.png"
 
-        fig.tight_layout()
```

Comment:
when started, tight_layout() seemed like a good idea. by now believe it disturbs more than it helps